tutorials/integrations/zapper.md

```diff
@@ -0,0 +1,105 @@
+
+This tutorial provides a detailed step-by-step guide for both users and developers to integrate with Zapper and maximize their experience within the Moonbeam ecosystem.
+
+## Get Started on Zapper
+
+You can navigate to [Zapper's official website](https://zapper.xyz){target=\_blank} and take the following steps to start interacting with the application:
+
+1. Click on the **Connect Wallet** button at the top right corner
+![Zapper main site](/images/tutorials/integrations/Zapper/zapper-1.webp)
+2. Select your preferred wallet from the supported options (e.g., MetaMask, Talisman, Coinbase Wallet)
+![Zapper wallet modal](/images/tutorials/integrations/Zapper/zapper-2.webp)
+3. Follow the on-screen prompts to connect securely to Zapper
+
+### Explore Your Portfolio
+
+Once your wallet is connected, Zapper will automatically detect and display your assets across supported networks, including Moonbeam. You can navigate to your main dashboard by clicking on **My profile** on the left sidebar or selecting it from the wallet icon on the top-right corner. The dashboard provides an overview of your holdings and activity split in the following categories:
+
+- **Summary**: displays your total assets, liabilities, and net worth across all networks
+- **Tokens**: lists all tokens held in your connected wallet, including GLMR (Moonbeam's native token) and any ERC-20 tokens
+- **DeFi**: shows active positions in Moonbeam-based DeFi platforms such as lending, liquidity pools, and staking
+- **NFTs**: displays any NFTs you own on Moonbeam or other supported chains
+- **Activity**: Zapper's dashboard provides real-time data, enabling efficient management of your on-chain activities
+
+![Zapper Dashboard](/images/tutorials/integrations/Zapper/zapper-3.webp)
+
+### Engage with Moonbeam's DeFi Ecosystem
+
+Zapper provides access to view the activity on Moonbeam's DeFi protocols, such as:
+
+- [**Moonwell**](https://moonwell.fi/){target=\_blank}: for accessing lending and borrowing services
+- [**StellaSwap**](https://app.stellaswap.com/exchange/swap){target=\_blank}: for participating in token swaps and liquidity provision by depositing token pairs into StellaSwap's liquidity pools
+- [**OmniLST**](https://omni.ls/){target=\_blank}: for engaging in liquid staking opportunities (stake GLMR tokens to receive liquid staking tokens and use them in other DeFi applications)
+- [**Prime Protocol**](https://www.primeprotocol.xyz/){target=\_blank}: for managing cross-chain assets and accessing borrowing services (deposit assets as collateral to borrow across multiple chains)
+
+You can follow these steps to review any of the aforementioned protocols:
+
+1. Select **Moonbeam** in the chain filter on the left sidebar. You'll be able to view the most trending DApps in the ecosystem and click on the protocol you want to explore 
```

Comment:
```suggestion
1. Select **Moonbeam** from the chain filter in the left sidebar. This will display the most trending dApps in the ecosystem. Click on the protocol you wish to explore
```

This slight change keeps it more action-oriented